mailserver.env

```diff
@@ -92,7 +92,7 @@ TLS_LEVEL=
 # 1 => Mail spoofing denied. Each user may only send with their own or their alias addresses. Addresses with extension delimiters(http://www.postfix.org/postconf.5.html#recipient_delimiter) are not able to send messages.
 SPOOF_PROTECTION=
 
-# Enables the Sender Rewriting Scheme. SRS is needed if your mail server acts as forwarder. See [postsrsd](https://github.com/roehling/postsrsd/blob/master/README.md#sender-rewriting-scheme-crash-course) for further explanation.
+# Enables the Sender Rewriting Scheme. SRS is needed if your mail server acts as forwarder. See [postsrsd](https://github.com/roehling/postsrsd/blob/main/README.rst) for further explanation.
```

Comment:
Technically the link was pointing to a README section to explain SRS, which is not present on the 2.x series README.

That old link now lives in the 1.x branch README:

https://github.com/roehling/postsrsd/blob/1.x/README.md#sender-rewriting-scheme-crash-course